src/test/resources/pages/instructorFeedbackDeleteSuccessful.html

```diff
@@ -967,6 +998,52 @@ <h5 class="col-xs-2" data-original-title="Select this option to use a custom tim
                 Submit
               </a>
             </div>
+            <div data-original-title="View the submitted responses for this feedback session" data-placement="top" data-toggle="tooltip" style="display: inline-block; padding-right: 5px;" title="">
+              <div class="btn-group margin-bottom-7px">
+                <a class="btn btn-default btn-xs btn-tm-actions session-view-for-test" href="/page/instructorFeedbackResultsPage?courseid=CFeedbackUiT.CS2104&fsname=private+session+of+characters1234567+%23&user=CFeedbackUiT.instructor">
+                  Results
+                </a>
+                <button aria-expanded="false" class="btn btn-default btn-xs btn-tm-actions dropdown-toggle session-results-options" data-toggle="dropdown" disabled="" type="button">
+                  <span class="caret">
+                  </span>
+                </button>
+                <ul class="dropdown-menu" role="menu">
+                  <li>
+                    <a class="session-view-for-test" href="/page/instructorFeedbackResultsPage?courseid=CFeedbackUiT.CS2104&fsname=private+session+of+characters1234567+%23&user=CFeedbackUiT.instructor">
+                      View Results
+                    </a>
+                  </li>
+                  <li>
+                    <a class="session-publish-for-test" data-fsname="private session of characters1234567 #" data-sending-published-email="true" href="/page/instructorFeedbackPublish?courseid=CFeedbackUiT.CS2104&fsname=private+session+of+characters1234567+%23&next=%2Fpage%2FinstructorFeedbackSessionsPage&user=CFeedbackUiT.instructor&token=${sessionToken}">
```

Comment:
I'm getting a local test failure too just like @leeyimin - `disabled=""` is expected but not present here. Unable to reproduce in CI though - really weird!